src/App.tsx

```diff
@@ -1,16 +1,27 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
+import { Movie } from './types/Movie';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => {
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd = (movie: Movie) => {
```

Comment:
```suggestion
  const handleAddMovie = (movie: Movie) => {
```

Use proper handler name.